portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
nit: Add period at the end of the sentence. Make comments completed sentences

Reply:
Done